packages/pagination/src/Pagination.stories.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import { ComponentStory } from '@storybook/react';
+
+import { storybookArgTypes } from '@leafygreen-ui/lib';
+
+import Pagination, { PaginationProps } from '.';
+
+export default {
+  title: 'Components/Pagination',
+  component: Pagination,
+  args: {
+    onCurrentPageOptionChange: undefined,
+  },
+  argTypes: {
+    currentPage: { control: 'number' },
+    numTotalItems: { control: 'number' },
+    itemsPerPage: { control: 'none' },
```

Comment:
To prevent the user from inputting an invalid value that would break the component. Like I mentioned in the other thread, there isn't a way through ts or prop-types to validate values that come in as props against other props

Reply:
Not to mention - there isn't really a good way to sync Storybook args with states. There's the `useArgs` hook, but it seems like there are still some issues with that.